min-image/runtimes/python/server.py

```diff
@@ -208,6 +208,7 @@ def main():
             file.write(pid)
             print(f'server.py: joined cgroup, close FD {fd_id}')
 
+    # drop privileges here
```

Comment:
Following the C code, I was thinking this would be one of the places where we would drop the privileges (either here or in `start_container()` just before forking (?) by invoking `ol.modify_caps(args)` (where args would have a list of capabilities we want to keep/ drop)